src/tech/v3/libs/smile/discrete_nb.clj

```diff
@@ -32,6 +32,10 @@
         p (count (-> feature-ds meta :count-vectorize-vocabulary :vocab->index-map))
         nb-model
         (case (options :discrete-naive-bayes-model)
+          :polyaurn DiscreteNaiveBayes$Model/POLYAURN
+          :wcnb DiscreteNaiveBayes$Model/WCNB
+          :cnb DiscreteNaiveBayes$Model/CNB
+          :twcnb DiscreteNaiveBayes$Model/TWCNB
```

Comment:
This is a build-in variant of tfidf.
https://github.com/haifengl/smile/blob/5ec63ae4c1e2b210131328054b6bf1e29bc30854/core/src/main/java/smile/classification/DiscreteNaiveBayes.java#L140

So we don't need a specific tfidf step in our pipeline for naive bayes with tfidf